engine/lib/phx/src/event_bus/payload.rs

```diff
@@ -311,10 +311,9 @@ impl EventPayload {
         value.iter().for_each(|v| f(*v));
     }
 
-    // TODO: uncomment when luajit_gen_ffi supports array/slices of strings properly
```

Comment:
Uncomment also a **TODO** part in `EventPayloadConverter:valueToPayload`. It relates to this one.